hack/jenkins/e2e.sh

```diff
@@ -39,9 +39,62 @@ function join_regex_no_empty() {
     fi
 }
 
+# Print an error message and exit with status 1.
+function error_exit() {
+  echo "$1" 1>&2
+  exit 1
+}
+
+# Retry a command up to a certain numbers.
```

Comment:
OK, I was doing similar things but then I realized I needed to write those boilerplate code at multiple places so I decided to make a function for it.